docs/application/dotnet/guides/connectivity/wifi.md

```diff
@@ -39,15 +39,16 @@ The main features of the Tizen.Network.WiFi namespace include:
 The application uses the infrastructure mode to connect to a wireless local area network (WLAN). The infrastructure mode requires a wireless access point. To connect to a WLAN, the application client must be configured to use the same service set identifier (SSID) as the access point.
 
 
-> **Note**  
+> **Note**
+>
 > You can test the Wi-Fi functionality on a target device only. The emulator does not support this feature.
 
 
 ## Prerequisites
 
 To enable your application to use the Wi-Fi functionality:
 
-1.  To use the [Tizen.Network.WiFi](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Network.WiFi.html) namespace, the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
+1.  To use the [Tizen.Network.WiFi](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Network.WiFi.html) namespace, the application has to request permission by adding the following privileges to the <b>`tizen-manifest.xml`</b> file:
```

Comment:
Please change <b>`tizen-manifest.xml`</b> to **<b>`tizen-manifest.xml`</b>**.
In markdown files, It is recommended to use **Bold**.
Please avoid usage of HTML tags.

Reply:
This was my suggestion for bold style. From now on wards, we will use this style. 
But as it is `tizen-manifest.xml`, no need to bold it.
I apologize for my mistake. Please can you remove bold style. 
`tizen-manifest.xml` is only needed.
Sorry, for the mistake.